generators/app/templates/.eslintrc

```diff
@@ -0,0 +1,9 @@
+---
+parser: babel-eslint
+
+env:
+  es6: true
+  amd: true
+  node: true
+  mocha: true
+  browser: true
```

Comment:
that used to be the case, but not anymore. We should publish our eslint config so that we can share it, or just copy it in here.